modules/visual_script/doc_classes/VisualScriptMathConstant.xml

```diff
@@ -4,7 +4,7 @@
 		Commonly used mathematical constants.
 	</brief_description>
 	<description>
-		Provides common math constants, such as Pi or Euler's constant, on an output Data port.
+		Provides common math constants, such as Pi, on an output Data port.
```

Comment:
This is okay. But could you let me know if the line 37-39 is also okay?
```
<constant name="MATH_CONSTANT_E" value="4" enum="MathConstant">
	Natural log: [code]2.718282[/code]
</constant>
```

Reply:
There seem to be several similar terms around _e_, I'm honestly not sure whether that part needs to be reworded or not. Not sure how that should be phrased.. maybe just "Mathematical constant _e_"?

Reply:
A quick Internet search for "Mathematical constant e" provides plenty of relevant information. I believe this phrasing is plenty useful and informative. We are to assume that Godot users have basic "Google-fu", after all.

@Noshyaar "Natural log: [code]2.718282[/code]" Well, natural log is a function, while E is a number. I would call it "Natural log base" or "Natural exponential base".